source/guides/mailers/basic-usage.md

```diff
@@ -24,7 +24,7 @@ end
 ```
 
 <p class="warning">
-  Both <code>.from</code> and <code>.to</code> informations MUST be specified when we deliver an email.
+  Both <code>.from</code> and <code>.to</code> MUST be specified when we deliver an email.
```

Comment:
I removed the word "informations" because it felt redundant. If you want me to add it back I can, but I'll add it as "information" because I believe that is the correct way to write this